### ISSUE TITLE
Navigation bar isTranslucent = false is not working correctly

### ISSUE THREAD

**Issue by wixvii** [andreamazz/AMScrollingNavbar#388]
this pod makes navigation translucent automatically but if you want to use it not translucent your collection views section header will be under navigationBar.

**Comment by prvnngrj** [comment]
Yes, sticky collection view headers don't work when navigation bar isTranslucent is set to false. When set to true, the section header also hides along with the navigation bar.

**Comment by prvnngrj** [comment]
@andreamazz
